style-guides/python.md

```diff
@@ -0,0 +1,32 @@
+# Python Infrastructure and Coding Standards 
+
```

Comment:
typo here, `organization`, also please break the line from `In addition.`

Reply:
Noted....